OrbitService/OrbitService.cpp

```diff
@@ -9,13 +9,16 @@
 #include "TcpServer.h"
 #include "TimerManager.h"
 
+// TODO: we should probably make it configurable
+constexpr uint32_t kAsioServerPort = 44766;
```

Comment:
Capture::GCapturePort = GParams.m_Port;

We should use the port in GParams as it is serialized and we can change it in the config.xml file.

Reply:
As discussed offline - the OrbitService does not really read config.xml - so it is always empty.